src/platform_impl/android/mod.rs

```diff
@@ -239,9 +239,26 @@ impl<T: 'static> EventLoop<T> {
                                             }
                                         }
                                     }
-                                    InputEvent::KeyEvent(_) => {
-                                        // TODO
-                                        handled = false;
+                                    InputEvent::KeyEvent(key) => {
+                                        let event = event::Event::WindowEvent {
+                                            window_id,
+                                            event: event::WindowEvent::KeyboardInput {
+                                                device_id,
+                                                input: event::KeyboardInput {
+                                                    scancode: key.key_code() as u32,
```

Comment:
Should use `scan_code` instead.
`key_code` would be more fitting for `virtual_keycode` below but can be left to `None` for now as well

Reply:
Done @msiglreith